### ISSUE TITLE
as3-universal-analytics implementation issue!

### ISSUE THREAD

**Issue by mihbht** [zwetan/as3-universal-analytics#31]
Hi Zwetan,

Thank you for posting this solution for tracking swf video files. I was going through the documentation for the API integration and I can follow what those steps. My issue is how to implement code inside the Flash file with a .swf format. I have found several tools to decompile the swf file but I'm not sure where to implement the code changes and how to recompile the file back and send events to Google Analytics. I would really appreciate your help. Can you please recommend me a tool to make my swf files to interact with Google Analytics?

Thank you,
Mihir

**Comment by zwetan** [comment]
Like any programming language out there, if you want to work with it you gonna have to learn it.

It is not our responsibility to teach you ActionScript, or how to compile a SWF, etc.
and there are a lot of resources out there, check out the Adobe Developer Network
there are literally hundred of tutorials.

This is not an implementation issue.
This is NOT A BUG.
issue closed

**Comment by mihbht** [comment]
Hi

I did not mean that your code had any issues, I just wanted to see if you had any implementation scenarios or any examples. I am checking those options with Adobe and I am not looking for a programming tutorial, just some hints as your documentation is sporadic and inconsistent. I am hoping for it to work as I have tried multiple solutions with no concrete results. Thank you for the help. Hope you have a nice day!

**Comment by zwetan** [comment]
wow documentation is "sporadic and inconsistent" ?

Wiki front page under [Documentation](https://github.com/zwetan/as3-universal-analytics/wiki#documentation)

there are **8 pages** of wiki documentation explaining it all in great details
you just gonna have to read it, something we can not do for you either

also in the library files you have a **documentation.zip**, unzip and open **index.html**
which not only document all the source code but also provide **usages** and **examples**

Before coming with insults when asking for help maybe do a minimum of homework

---

![uanalytics_documentation1](https://cloud.githubusercontent.com/assets/236154/18371937/b65c00c4-7638-11e6-880b-1ee8f8bad7a0.png)

---

![uanalytics_documentation2](https://cloud.githubusercontent.com/assets/236154/18371949/c8ee4904-7638-11e6-93da-0da9c0a7c7fa.png)

---

![uanalytics_documentation3](https://cloud.githubusercontent.com/assets/236154/18371958/d1d6e468-7638-11e6-8d5f-d93b62eed84d.png)

**Comment by mihbht** [comment]
Hey

I did not mean to insult you in any way. I only said what I felt; you are not completely aware of what I am trying to achieve and it's not your fault. Thanks again for the additional help. I am aware of what I might have to do for my solution, and my only hope now is that your solution should work, which I think it probably should. I am new to programming and analytics and everyday is a learning experience for me; if you are a veteran programmer I feel bad if I offended you in any way- That was **never** my intention. I will update you for sure if your solution is applicable in my case and also in case if I find a better solution. Thank you.
